contracts/src/interfaces/IEETHHyperdrive.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IHyperdrive } from "./IHyperdrive.sol";
+
+interface IEETHHyperdrive is IHyperdrive {
+    /// @notice Gets the vault used as this pool's yield source.
+    /// @return The compatible yield source.
+    function vault() external view returns (address);
+
+    /// @notice Gets the EtherFi oracle contract.
+    /// @return The EtherFi oracle contract.
+    function oracle() external view returns (address);
```

Comment:
Are these functions actually implemented? I don't see them in EETHTarget0. Also, shouldn't we only have `vault` (or `liquidityPool` in this case? In the case of Aave, we rename `pool` to `vault`, so I'd consider renaming this here too.